### PR TITLE
Pages & doctest set up

### PR DESCRIPTION
**Pull request by Tianyu9748**
Already set up pages and doctest. All files will be revised in the next pull request.

### CODE REVIEW

**Comment by Tianyu9748** [comment]
Set up pages and doctest. 
TODO: need to fix name to raptor

**Comment by Tianyu9748** [comment]
Set up Pages & Doctest

File: raptors/src/executor.rs [hunk 1]
```diff
@@ -34,4 +44,4 @@ mod tests {
         exec.compute(load);
         assert!(now.elapsed() >= time::Duration::from_millis(16));
     }
-}
+}
```
Comment (codes1gn) [inline raptors/src/executor.rs]: add new line

File: raptors/src/lib.rs [hunk 2]
```diff
@@ -9,4 +9,4 @@ mod tests {
         let result = 2 + 2;
         assert_eq!(result, 4);
     }
-}
+}
```
Comment (codes1gn) [inline raptors/src/lib.rs]: ditto

File: raptors/src/messages.rs [hunk 3]
```diff
@@ -71,4 +79,4 @@ mod tests {
         load.mock_run();
         assert!(now.elapsed() >= time::Duration::from_millis(16));
     }
-}
+}
```
Comment (codes1gn) [inline raptors/src/messages.rs]: ditto

**Comment by Tianyu9748** [comment]
Format fixed

**Review by codes1gn — APPROVED** [review]

**Pull request merged**